tests/src/bristol_contraflow_cycleway/geometry.json

```diff
@@ -2263,7 +2263,7 @@
         "type": "Polygon"
       },
       "properties": {
-        "control": "Signalled",
+        "control": "Signed",
```

Comment:
This previously was marking the footway crossing just north of https://www.openstreetmap.org/node/21310514 as signalized. This is incorrect; there's actually a pelican crossing at the traffic signal node, but the crossing isn't mapped. There's an unrelated footpath just to the north unrelated to the signal